src/instance.ts

```diff
@@ -14,7 +16,7 @@ export class ExtensionInstance {
 
 	context: ExtensionContext;
 
-	networkComponent: NetworkComponent;
+	networkComponent?: NetworkComponent;
```

Comment:
networkComponent is handled by our code, why would it ever be in question if it's added to the plugin instance or not? 

Either it's added or it's not.

Reply:
networkComponent is initially `undefined`, it is only populated once the child process has instantiated.